Makefile

```diff
@@ -255,7 +255,7 @@ db-clean: ## create a new, clean database using the script file in the database.
 
 db-seed: ## create a new, database seeded with test data using the script file in the database. defaults to using the folder specified in database/mssql/.env, but can be overriden with n=PSP_PIMS_S15_00.
 	@echo "$(P) Seed the database with test data. n=FOLDER_NAME (PSP_PIMS_S15_00)"
-	TARGET_SPRINT=$(n) SEED=TRUE docker-compose up -d --build database;
+	TARGET_SPRINT=$(n) SEED=TRUE docker-compose up -d --build --force-recreate database;
```

Comment:
--force-recreate ensures that a pre-existing database can't interfere with a seed.

Reply:
good, i've been doing a seed on the previous just to force it